src/components/ColorChoiceDialog.js

```diff
@@ -10,6 +10,8 @@ import DialogContent from "@material-ui/core/DialogContent";
 import DialogTitle from "@material-ui/core/DialogTitle";
 import { ChromePicker } from "react-color";
 import ResponsiveSetCard from "./ResponsiveSetCard";
+import { darkTheme } from "../themes";
```

Comment:
This can be combined into a single import.

Reply:
Lol this actually occurred to me after I did it and I totally forgot to change it... thank you!

Reply:
Pushed fix